### ISSUE TITLE
Warnings Change

### ISSUE THREAD

**Issue by cjk12** [convexopt/gpkit#163]
Dont know if this is a mosek thing, but it would be great if:

```python
Warning: The variable with index '12' has only positive coefficients akj.
 The problem is possibly ill-posed.
```

actually told me the variable it is having a problem with rather than an index.

**Comment by bqpd** [comment]
Pretty sure you should be seeing "$VAR is missing a lower bound" in that
case as well. If not try:

gp.genA()
gp.checkbounds()

**Comment by cjk12** [comment]
Yea, so I pulled and this is updated.  It would be nice if I could also
turn off the warnings such as that posted above however...  Right now it is
printing both.

On Wed, Feb 18, 2015 at 3:32 PM, bqpd <notifications@github.com> wrote:

> Pretty sure you should be seeing "$VAR is missing a lower bound" in that
> case as well. If not try:
>
> gp.genA()
> gp.checkbounds()
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/convexopt/gpkit/issues/163#issuecomment-74942099>.
>



-- 
*Cody J. Karcher*
Graduate Student
Department of Aeronautics and Astronautics
Massachusetts Institute of Technology

**Comment by bqpd** [comment]
run it as gp.solve(printing=False) :)

**Comment by bqpd** [comment]
Oh and if your're seeing the MOSEK warnings that's a MOSEK bug. Probably
another section of their code that doesn't properly print to the stream
buffer.

**Comment by cjk12** [comment]
I was printing on False, so I dont think that's on my end.  May be a Mosek
bug.  I have no idea how to check that out.

On Wed, Feb 18, 2015 at 3:41 PM, bqpd <notifications@github.com> wrote:

> Oh and if your're seeing the MOSEK warnings that's a MOSEK bug. Probably
> another section of their code that doesn't properly print to the stream
> buffer.
>
>
> On Wed, Feb 18, 2015 at 3:39 PM, Edward Burnell <eburn@mit.edu> wrote:
>
> > run it as gp.solve(printing=False) :)
> >
> > On Wed, Feb 18, 2015 at 3:34 PM, cjk12 <notifications@github.com> wrote:
> >
> >> Yea, so I pulled and this is updated. It would be nice if I could also
> >> turn off the warnings such as that posted above however... Right now it
> is
> >> printing both.
> >>
> >>
> >> On Wed, Feb 18, 2015 at 3:32 PM, bqpd <notifications@github.com> wrote:
> >>
> >> > Pretty sure you should be seeing "$VAR is missing a lower bound" in
> that
> >> > case as well. If not try:
> >> >
> >> > gp.genA()
> >> > gp.checkbounds()
> >> >
> >> > —
> >> > Reply to this email directly or view it on GitHub
> >> > <https://github.com/convexopt/gpkit/issues/163#issuecomment-74942099
> >.
> >> >
> >>
> >>
> >>
> >> --
> >> *Cody J. Karcher*
> >> Graduate Student
> >> Department of Aeronautics and Astronautics
> >> Massachusetts Institute of Technology
> >>
> >> —
> >> Reply to this email directly or view it on GitHub
> >> <https://github.com/convexopt/gpkit/issues/163#issuecomment-74942464>.
> >>
> >
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/convexopt/gpkit/issues/163#issuecomment-74943514>.
>



-- 
*Cody J. Karcher*
Graduate Student
Department of Aeronautics and Astronautics
Massachusetts Institute of Technology

**Comment by whoburg** [comment]
@cjk12, can you please re-paste the offending warning message here, now that you have pulled?

**Comment by cjk12** [comment]
```python
P_climb has no lower bound
Pi_climb has no upper bound
Warning: The variable with index '130' has only positive coefficients akj.
 The problem is possibly ill-posed.
.
Warning: The variable with index '461' has only negative coefficients akj.
 The problem is possibly ill-posed.
```

Seems a little redundant?

**Comment by whoburg** [comment]
Thanks. That indeed looks like a mosek bug. I imagine @bqpd can hack their code again, but this is low priority for right now. Thanks for pointing it out!

**Comment by bqpd** [comment]
Fixed in 2693c456472c3bfbcc643d5d94e1e9fec2c5d3ff
